### PR TITLE
[BUG] v2.8.1 with a Hosted IE environment fails to initialize for a hosted instance of IE #1822

### PR DESCRIPTION
**Pull request by MSNev**
[BUG] IE8 Support was broken by several components #1823

### CODE REVIEW

File: AISKU/src/ApplicationInsightsDeprecated.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ export class AppInsightsDeprecated implements IAppInsightsDeprecated {
         // Note: This must be called before loadAppInsights is called
         proxyAssign(snippet, this, (name: string) => {
             // Not excluding names prefixed with "_" as we need to proxy some functions like _onError
-            return name && _ignoreUpdateSnippetProperties.indexOf(name) === -1;
+            return name && arrIndexOf(_ignoreUpdateSnippetProperties, name) === -1;
```
Comment (MSNev) [inline AISKU/src/ApplicationInsightsDeprecated.ts:190]: .indexOf() doesn't exist on IE8 :-(

File: AISKULight/package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
         "tslib": "*"
     },
     "dependencies": {
-        "@microsoft/dynamicproto-js": "^1.1.4",
+        "@microsoft/dynamicproto-js": "^1.1.5",
```
Comment (MSNev) [inline AISKULight/package.json:52]: This is the other half of fixing IE8

File: shared/AppInsightsCore/src/JavaScriptSDK/InstrumentHooks.ts [hunk 3]
```diff
@@ -266,9 +266,9 @@ export function InstrumentFuncs(target:any, funcNames:string[], callbacks: IInst
  * @param callbacks - The callbacks to configure and call whenever the function is called
  * @param checkPrototype - If the function doesn't exist on the target should it attempt to hook the prototype function
  */
-export function InstrumentEvent(target: any, evtName: string, callbacks: IInstrumentHooksCallbacks, checkPrototype:boolean = true): IInstrumentHook {
+export function InstrumentEvent(target: any, evtName: string, callbacks: IInstrumentHooksCallbacks, checkPrototype?: boolean): IInstrumentHook {
     if (target && evtName && callbacks) {
-        let owner = _getOwner(target, evtName, checkPrototype);
+        let owner = _getOwner(target, evtName, checkPrototype) || target;
```
Comment (MSNev) [inline shared/AppInsightsCore/src/JavaScriptSDK/InstrumentHooks.ts:271]: This is part of the primary fix for all of IE

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -5,9 +5,9 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master, beta ]
+    branches: [ master, beta, Release2.7 ]
```
Comment (MSNev) [inline .github/workflows/ci.yml:8]: Branch Release2.7 is for the 2.7.x release (to also fix IE8 for the 2.7.4 with a 2.7.5 release)

**Review by Karlie-777 — APPROVED** [review]

**Pull request merged**